pymongo/encryption.py

```diff
@@ -314,29 +348,6 @@ def close(self):
         self._closed = True
         self._auto_encrypter.close()
```

Comment:
I suggest closing the client here (if we created it):
```python
if self._internal_client:
    self._internal_client.close()
    self._internal_client = None
```

Reply:
Done.